WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -3416,12 +3416,27 @@ private bool CreateArray()
     /// <returns>ChunkViewModel or null</returns>
     public ChunkViewModel? GetChildNode(int index)
     {
-        if (!IsArray || Properties is not ObservableCollectionExtended<ChunkViewModel> children ||
-            index >= children.Count)
+        if (!IsArray)
         {
             return null;
         }
 
+        if (Properties.Count == 0)
+        {
+            CalculateProperties();
+        }
+
+        if (Properties is not ObservableCollectionExtended<ChunkViewModel> children)
+        {
+            return null;
+        }
+
+        if (index < children.Count)
```

Comment:
I think you meant to return null here.

Reply:
?

Reply:
The original check is if greater than or equal and returns null, your check doesn't do this anymore

Reply:
Right! Thanks, fixed